src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Have `ServiceLocator` implement `ServiceCollectionInterface`
  * Add `#[Lazy]` attribute as shortcut for `#[Autowire(lazy: [bool|string])]` and `#[Autoconfigure(lazy: [bool|string])]`
  * Add `#[AutowireMethodOf]` attribute to autowire a method of a service as a callable
+ * `ContainerBuilder::registerAttributeForAutoconfiguration()` is applied to child classes for the autoconfigured attribute, except there is another autoconfiguration for this child attribute class.
```

Comment:
```suggestion
 * Make `ContainerBuilder::registerAttributeForAutoconfiguration()` propagate to child attribute classes
```

Reply:
From a developer POV: "Allows extending attribute classes registered with `ContainerBuilder::registerAttributeForAutoconfiguration()` and propagate to the child attribute classes when it doesn't have its own autoconfiguration."